internal/manager/run.go

```diff
@@ -162,6 +162,38 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic) e
 		return err
 	}
 
+	if featureGates[gatewayFeature] {
+		setupLog.Info("Gateway API Feature Enabled, Checking Existence of CRDs")
+		crdExistsCheckers := []crdExistsChecker{
+			gatewayCRDExistsChecker,
+			gatewayClassCRDExistsChecker,
+			httpRouteCRDExistsChecker,
+		}
```

Comment:
Perhaps we can place that outside this `func` just below definitions of all the `crdExistsChecker`?

Similar to the alpha crds checker list below

Reply:
you mean we define 2 lists `gatewayBetaCRDExistsCheckers` and `gatewayAlphaCRDExistsCheckers`?

Reply:
> I think it would be clearer to know that the checkers are used for checking beta APIs and alpha APIs.

Yup, that's my thought as well. I just feel that we can move this outside of this `func`  with proper names (the ones you suggested are perfect).

Reply:
updated.